Upstream: yes
From 6cdabc4dd0c6c804a093b94c269461ce376fed4f Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Mon, 21 Jan 2019 10:47:09 +0000
Subject: [PATCH] Drop the autoptr definition for ESource

Evolution-data-server includes the autoptr definition for ESource.

See: GNOME/evolution-data-server#73

Without this change, GNOME Todo does not build on the Continuous CI
infrastructure.

Signed-off-by: Emmanuele Bassi <ebassi@gnome.org>
---
 plugins/eds/gtd-eds-autoptr.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plugins/eds/gtd-eds-autoptr.h b/plugins/eds/gtd-eds-autoptr.h
index eb9b011..42709ee 100644
--- plugins/eds/gtd-eds-autoptr.h
+++ plugins/eds/gtd-eds-autoptr.h
@@ -25,4 +25,3 @@
 G_DEFINE_AUTOPTR_CLEANUP_FUNC (ECalComponent, g_object_unref);
 G_DEFINE_AUTOPTR_CLEANUP_FUNC (ECalComponentId, e_cal_component_free_id);
 G_DEFINE_AUTOPTR_CLEANUP_FUNC (ECalClient, g_object_unref);
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (ESource, g_object_unref);
-- 
2.18.1

